.github/workflows/ci.yaml

```diff
@@ -52,38 +52,38 @@ jobs:
           composer remove --no-update --dev --no-interaction --no-progress \
             phan/phan phpstan/phpstan vimeo/psalm \
             infection/infection friendsofphp/php-cs-fixer
-          composer update --prefer-dist --no-interaction --no-progress ${{ matrix.dependencies }}
+          composer update --prefer-dist --no-interaction --no-progress
 
       - name: Execute tests
         run: |
-          php vendor/bin/phpunit
+          ./vendor/bin/phpunit
```

Comment:
`phpunit` binary should be executable.

Reply:
It gets crazy in Windows, so I'd prefer the existing variant which, to my best knowledge, works everywhere.

Reply:
Further investigation in https://github.com/szepeviktor/phpunit-vs-windows-for-sanmai